### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by XhmikosR**
1. Removed unused IDs
2. Add missing page description to index.html
3. Remove unneeded quotes
4. Remove the unused package.json

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
     </div>
   </div>
 </div>
-<div id="content">
+<div>
 
 {{ content }}
 
-</div>
+</div> <!-- /content -->
```
Comment (benbalter) [inline _layouts/home.html]: Looks like you've removed the ID but then added a comment referencing that ID?

**Comment by benbalter** [comment]
@XhmikosR per the other pull requests, as much as these can be broken into individual changes, it'd be much easier to review and merge, but thanks again for being so thorough.

File: _posts/2013-10-01-modern-approach-to-open-data.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Developers from the [Sunlight Foundation](http://sunlightfoundation.com/), [GovT
...
+But it didn't end there. Their efforts were about more than just code — [it was about making government transparency a group effort](http://sunlightfoundation.com/blog/2013/08/20/a-modern-approach-to-open-data/). A community began to emerge around the shared challenges of government data. Subject matter experts shed light on [the nuances of government datasets](https://github.com/unitedstates/congress-legislators/issues/2), and crowd source began to [fill in the gaps of missing data](https://github.com/unitedstates/congress-legislators/pull/39) while others still took to building tools to [analyze and visualize those very datasets](http://news.yahoo.com/senate-budget-amendments-interactive-track-changes-153640966.html).
```
Comment (benbalter) [inline _posts/2013-10-01-modern-approach-to-open-data.md:16]: What changed here?

**Comment by XhmikosR** [comment]
I'll have a look tomorrow.

File: _posts/2013-10-01-modern-approach-to-open-data.md (hunk 2 above)
Comment (XhmikosR) [inline _posts/2013-10-01-modern-approach-to-open-data.md:16]: It's the typo fix there->their

**Comment by XhmikosR** [comment]
Should be ok now.

**Comment by XhmikosR** [comment]
I split the PR. Seriously, this is a lot more time consuming than simply one to look at the changes but anyway...

**Pull request merged**